app/src/main/java/com/example/aboutme/MyName.kt

```diff
@@ -0,0 +1,3 @@
+package com.example.aboutme
+
+data class MyName(var name: String = "", var nickname: String = "")
```

Comment:
Here you can see that is a sign with red at the end of the file. 

Github know that on each file there should be a new empty line.